### ISSUE TITLE
double backlog send to slack

### ISSUE THREAD

**Issue by judika03** [omise/graylog-plugin-slack-notification#4]
Hi sir,

I have problem that backlog same alert message sent to slack is double.

**Comment by KongZ** [comment]
Check out `Alerts & Events` menu. The notification will process message that trigger from alerts. If you see 2 notifications that mean it is actually 2 alerts.

In case there are multiple alerts and you want to notify only one, you can set the backlog item to 1.

**Comment by judika03** [comment]
I already to set backlog to 1, but I have recieve in slack to 2 same message alert. I check out Alerts & Events menu, there is 1 alert but sending to slack 2 alert double same.

**Comment by KongZ** [comment]
That is weird. You may need to modify event grace period as well.
The plugin process whatever alerts sending message to notification. I'm pretty sure that it trigger twice so plugin send two slack messages.

**Comment by judika03** [comment]
Of course, I already to setting up grace period to 5 minute,  Alerts are still sending double same alert. Actually alert send to same alert with same timestamp.
